src/features/ModelPlan/ModelToOperations/_components/OptionPanel/index.tsx

```diff
@@ -152,9 +157,14 @@ const MTOOptionsPanel = () => {
         className="margin-top-2"
       >
         <div
-          style={{ paddingTop: '2px', paddingBottom: '2px' }}
+          style={{
+            paddingTop: '2px',
+            paddingBottom: '2px',
+            borderTopLeftRadius: '6px',
+            borderTopRightRadius: '6px'
+          }}
           className={classNames(
-            'display-flex flex-justify bg-base-lightest padding-x-3 radius-top-lg bg-base-lighter'
+            'display-flex flex-justify bg-base-lightest padding-x-3 bg-base-lighter'
```

Comment:
The previous padding of `radius-top-lg` was correct, as it matches the pixels of Figma - .5rem or 8px 
<img width="178" alt="Screenshot 2025-05-16 at 7 33 07 AM" src="https://github.com/user-attachments/assets/3d4fa891-cc9f-464b-912c-2eef9d8530d7" />

I think there's an issue with the layering of a div with a border-radius on top of another div with a border-radius.  I found the extended the border-radius of the parent card radius from .5rem to .65rem on the top two corners better aligns those border

USWDS also has a utility class for padding 2 px - You can you `padding-y-2px` instead of setting the style manually.

Reply:
Yeah the issue is the nesting border radius, since border radius has both inside and outside, when we make the outside radius to be 8px/.5rem, the inside radius is really 8px-2px border = 6px. I guess it depends on how we translate the figma, 'does the 8px apply to outside radius or inside?'. I agree with you on the .65rem looks better aligned so let's say the 8px radius is for inside lol
the padding code was leftover, but yes I can update that!